app-config.yaml

```diff
@@ -76,6 +76,13 @@ integrations:
     - host: dev.azure.com
       token: ${AZURE_TOKEN}
 
+  # Plugin: Bitbucket Server
+  bitbucketServer:
+    - host: ${BITBUCKET_SERVER_HOST}
+      apiBaseUrl: ${BITBUCKET_API_BASE_URL}
+      username: ${BITBUCKET_SERVER_USERNAME}
+      password: ${BITBUCKET_SERVER_PASSWORD}
```

Comment:
Please define these values in [`app-config.example.yaml`](https://github.com/janus-idp/backstage-showcase/blob/b458fb13e45429b8b28b9a3de6894d929b501c92/app-config.example.yaml#L33) as well. Without it the instance refuses to start locally:

```
$ yarn start
yarn run v1.22.19
$ turbo run start --parallel
• Packages in scope: app, backend
• Running start in 2 packages
• Remote caching disabled
app:start: cache bypass, force executing 7817bfd2efabd9a9
backend:start: cache bypass, force executing bf6ab47618c2b7ff
app:start: $ backstage-cli package start
backend:start: $ backstage-cli package start
backend:start: Build succeeded
app:start: Loaded config from app-config.yaml, app-config.local.yaml
app:start: <i> [webpack-dev-server] Project is running at:
app:start: <i> [webpack-dev-server] Loopback: http://localhost:3000/, http://[::1]:3000/
app:start: <i> [webpack-dev-server] Content not from webpack is served from '/Users/tcoufal/Programming/janus-idp/backstage-showcase/packages/app/public' directory
app:start: <i> [webpack-dev-server] 404s will fallback to '/index.html'
app:start: <i> [webpack-dev-middleware] wait until bundle finished: /
backend:start: Loaded config from app-config.yaml, app-config.local.yaml
backend:start: 2023-06-07T12:10:10.537Z backstage info Found 5 new secrets in config that will be redacted
backend:start: Backend failed to start up Error: Missing required config value at 'integrations.bitbucketServer[0].host'
backend:start:     at ConfigReader.getString (/Users/tcoufal/Programming/janus-idp/backstage-showcase/node_modules/@backstage/config/dist/index.cjs.js:240:13)
backend:start:     at readBitbucketServerIntegrationConfig (/Users/tcoufal/Programming/janus-idp/backstage-showcase/node_modules/@backstage/integration/dist/index.cjs.js:748:23)
backend:start:     at Array.map (<anonymous>)
backend:start:     at readBitbucketServerIntegrationConfigs (/Users/tcoufal/Programming/janus-idp/backstage-showcase/node_modules/@backstage/integration/dist/index.cjs.js:772:18)
backend:start:     at BitbucketServerIntegration.factory (/Users/tcoufal/Programming/janus-idp/backstage-showcase/node_modules/@backstage/integration/dist/index.cjs.js:809:19)
backend:start:     at ScmIntegrations.fromConfig (/Users/tcoufal/Programming/janus-idp/backstage-showcase/node_modules/@backstage/integration/dist/index.cjs.js:1697:51)
backend:start:     at AzureUrlReader.factory (/Users/tcoufal/Programming/janus-idp/backstage-showcase/node_modules/@backstage/backend-common/dist/index.cjs.js:1397:52)
backend:start:     at UrlReaders.create (/Users/tcoufal/Programming/janus-idp/backstage-showcase/node_modules/@backstage/backend-common/dist/index.cjs.js:3673:22)
backend:start:     at UrlReaders.default (/Users/tcoufal/Programming/janus-idp/backstage-showcase/node_modules/@backstage/backend-common/dist/index.cjs.js:3688:23)
backend:start:     at makeCreateEnv (webpack-internal:///./src/index.ts:55:93)
backend:start:     at main (webpack-internal:///./src/index.ts:112:23)
```